apps/native-ui-storybook/components/Heading/Variants.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Box, Center, Text, Heading, VStack } from '@utilitywarehouse/native-ui';
+
+const TextWrap = ({ title, children }: any) => (
+  <Box gap="$2">
+    <Text textTransform="uppercase" size="xs" highlight color="$grey600">
```

Comment:
I find these 2 sizes conventions a bit confusing 🤔 because they are so different from each other. 
Technically Heading could only have h1', 'h2', 'h3', 'h4', 'h5', 'h6' but what if I add these to Text?

Reply:
These are just presentational for the docs to mimic this:
![Screenshot 2024-07-04 at 17 43 10](https://github.com/utilitywarehouse/design-systems/assets/7591840/ffee7b7e-7cf9-42e4-a0ba-ca4aba0552d4)

Reply:
If you add them to text they won't work... wait until docs are built and you'll see :P